### DISCUSSION TITLE
Whisper cuts timestamp every seconds

### DISCUSSION THREAD

**Discussion by 701789262a** [openai/whisper#1206] [General]
Using this code results in having every timestamp trimmed in seconds (the segment start and ends ONLY with integers)

```
model = whisper.load_model("large-v2")

options = dict(without_timestamps=False)
transcribe_opt = dict(task="transcribe",**options)
result = model.transcribe("audio.wav",**transcribe_opt)
```

I've seen that it's possible to use stable-ts but I was wondering if was missing something (as if stable-ts only optimizes it but whisper itself should be able to return decimal timestamp)

**Comment by jongwook** [comment]
The model has a bias on integer timestamps. Specifying `word_timestamps=True` will recalculate the timestamps based on the predicted word-level timestamps, and it tends to be more accurate.

Related:

- #435
- #809 
- #1122
